packages/forms-web-app/src/middleware/cache-busting.js

```diff
@@ -0,0 +1,9 @@
+const { isProduction } = require('../config');
+const version = require('../../package.json').version;
```

Comment:
does it get updated?

Reply:
Will use the timestamp of when the application starts for now